18/homework_18.3.py

```diff
@@ -0,0 +1,7 @@
+
+class MyStr(str):
+    def __str__(self):
+        return super().__str__().upper()
```

Comment:
This can be simplified to `return self.upper()` since `self` is already a string because the class in inherited from `str`